Controls/DataControls/DataControls.UWP/ListView/View/Controls/RadListViewItem.IDragDropElement.cs

```diff
@@ -267,11 +267,36 @@ void IDragDropElement.OnDragDropComplete(DragCompleteContext context)
                 this.FinalizeReorder(context);
 
                 var isExecuted = false;
+                var destinationItem = default(RadListViewItem);
```

Comment:
I believe it is more readable to make it` RadListViewItem destinationItem = null;`
I am OK with default as well .. it is only a suggestion .. if you prefer default I am fine :)